pymc_marketing/mmm/base.py

```diff
@@ -1126,7 +1127,14 @@ def plot_grouped_contribution_breakdown_over_time(
         area_params = dict(stacked=True, ax=ax)
         if area_kwargs is not None:
             area_params.update(area_kwargs)
-        all_contributions_over_time.plot.area(**area_params)
+        try:
+            all_contributions_over_time.plot.area(**area_params)
+        except ValueError:
+            warnings.warn(
+                "Each contribution value must be either all positive or all negative",
+                stacklevel=2,
+            )
+            return None
```

Comment:
I do not think there is an easy fix. It is a limitation of the plot. 

Any other ideas?

Reply:
This is what comes to my mind. We were basically just lucky that posterior estimates where positive for "Base" group.

Reply:
I would just return the figure still